server/routes.js

```diff
@@ -2,19 +2,25 @@ var controller = require('./controllers');
 var router = require('express').Router();
 
 // GET /api/listings?query=
-router.get('/listings', controller.getListings);
+router.get('/api/products', controller.getProducts);
```

Comment:
The /api/ portion of the route should have already been handled in server.js

Reply:
Can you update the function in server.js to be /api/pl (to represent products & listings API) and then all the URLs in the routes.js file can be /listings, /products, etc

Reply:
Yep got it.